src/ess/reduce/nexus/workflow.py

```diff
@@ -513,9 +513,18 @@ def assemble_monitor_data(
 
 def parse_disk_choppers(
     choppers: AllNeXusComponents[snx.NXdisk_chopper, RunType],
-) -> Choppers[RunType]:
-    """Convert the NeXus representation of a chopper to ours."""
-    return Choppers[RunType](
+) -> RawChoppers[RunType]:
```

Comment:
> > > The tof workflow incorrectly assumes that loaded Choppers are DiskChopper objects.
> >
> > So how did the workflow work so far? Have chopper params always been set by hand (using the "incorrect" type)?
>
> I think so. I don't see any other way that would have worked. In particular for DREAM, we never (to my knowledge) used nexus files, only a hard coded list of choppers.

I think this is correct, we read and used choppers from nexus for the TBL but in that file, the list of choppers was empty, so we then just built a LUT without choppers.